src/components/GitmojiList/Toolbar/ListModeSelector/styles.module.css

```diff
@@ -6,10 +6,10 @@
   align-items: center;
   border-radius: 4px;
   border: none;
-  color: var(--text);
+  color: var(--cardText);
```

Comment:
I think we should revert this change as now the text in light mode is a bit more light and difficult to see:

<img width="197" alt="Screenshot 2022-12-22 at 19 59 38" src="https://user-images.githubusercontent.com/7629661/209207469-c3147279-8bf2-4f00-a9ee-5ce314b6a62b.png">
<img width="195" alt="Screenshot 2022-12-22 at 19 59 44" src="https://user-images.githubusercontent.com/7629661/209207471-dfb39262-abdf-4a3d-bd90-54fe2b8343e1.png">

```suggestion
  color: var(--text);
```

Reply:
`--text` does not exist. So should I delete the line or create the var ?

Reply:
Let's leave it like this, let's revert the change to `--text` and I'll fix the places where it's being used as an unset variable on another PR! 🙏🏼